templates/origami-manifest.js

```diff
@@ -0,0 +1,52 @@
+const details = answers => {
+	// TODO why don't arrays render here!?
+	return `{
+	"description": "${answers.description}",
+	"keywords": ${answers.keywords},
+	"origamiType": "module",
+	"origamiCategory": "components",
+	"origamiVersion": 1,
+	"brands": ${answers.brands},
+	"support": "https://github.com/Financial-Times/${answers.name}/issues",
+	"supportContact": {
+		"email": "${answers.email}",
+		"slack": "financialtimes/${answers.slack}"
+	},
+	"supportStatus": "${answers.status}",
+	"browserFeatures": {},
+
+	"ci": {
```

Comment:
Should we remove this section? I can't remember what we decided on with the "ci" section.